docs/general/gotchas.md

```diff
@@ -21,3 +26,51 @@ import 'file?name=[name].[ext]!../img/yourimg.png';
 ```
 
 Then webpack will correctly transfer the image to the build folder.
+
+## Load reducers optimistically<a name="optimistic-reducers"></a>
+
+If you have some `containers` that aren't _route specific_ and that should be available throughout the app, like a `NavigationBar`, you need to add their respective reducers to the root reducer with the help of `combineReducers`.
+
+``` js
+// In app/reducers.js
+
+...
+import { combineReducers } from 'redux-immutable';
+...
+
+import navigationBarReducer from 'containers/NavigationBar/reducer';
+
+export default combineReducers({
+  route: routeReducer,
+  global: globalReducer,
+  language: languageProviderReducer,
+  navigationBar: navigationBarReducer,
+  ...asyncReducers,
+});
+```
+
+## Exclude modules from Babel processing<a name="exclude-modules"></a>
+
+Excluding modules from babel processing has never been so easy! Just add the package name to `exclude` array in `internals/config.js` and you're all set!
+
+``` js
+// in internals/config.js
+
+exclude: [
+  'chalk',
+  'compression',
+  'cross-env',
+  'express',
+  'ip',
+  'minimist',
+  'sanitize.css',
+  'your-unwanted-package', <- add your-unwanted-package
+  ...
+]
+```
+
+## Running tests in `watch` mode<a name="watch-test"></a>
```

Comment:
These `<a name` tags are unnecessary, GitHub automatically assigns an `id` to all headings so you can deep link to them! 😊